<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerWindow</class>
 <widget class="QMainWindow" name="ServerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>591</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>591</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>591</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="closeServer">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>540</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Close server</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="chatSendLE">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>561</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="chatWindow">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>561</width>
      <height>151</height>
     </rect>
    </property>
    <property name="plainText">
     <string>This is the famous chat everybody talks about</string>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QListWidget" name="playersList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="StartGameBtn">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>490</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Game</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
